File: C/Ex14.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float a, b, c;

    printf("Digite o primeiro numero: ");
    scanf("%f", &a);
    printf("Digite o segundo numero: ");
    scanf("%f", &b);
    printf("Digite o terceiro numero: ");
    scanf("%f", &c);

    if (a>=b && a>=c) {
        printf("O maior numero = %.2f", a);
        }
    else if(b>=a && b>=c) {
        printf("O maior numero = %.2f", b);
        }
    else if(c>a && c>b) {
        printf("O maior numero = %.2f", c);
        }

        return 0;

        }    

File: Python/Ex14.py
a= float(input("Digite o primeiro valor: "))
b= float(input("Digite o segundo valor: "))
c= float(input("Digite o terceiro valor: "))
if (a>=b) and (a>=c):
    print(f"O maior numero = {a:.2f}")
elif (b>=a) and (b>=c):
    print(f"O maior numero = {b:.2f}") 
elif (c>a) and (c>b):
    print(f"O maior numero = {c:.2f}")         